### PR TITLE
Remove Bento helpers, 2nd part

### PR DESCRIPTION
**Pull request by eduardoj**
Remove helpers only used in the old UI, from `p` till `r`, located under `src/api/app/helpers/webui/`.

~~Some refactoring was also done, moving methods only used in its class to private,~~

### CODE REVIEW

**Comment by eduardoj** [comment]
[Remove bento helpers](https://trello.com/c/aaNWKJZf/1240-remove-bento-helpers)

**Comment by eduardoj** [comment]
Marking as "DO NOT MERGE" until fix spider tests are fixed.

**Comment by eduardoj** [comment]
Spider tests are fixed! :tada: Removing "DO NOT MERGE" label.

**Comment by eduardoj** [comment]
> Please clean up the commit message of [8cb2016](https://github.com/openSUSE/open-build-service/commit/8cb201636ee3d39851a8eedc7e60f86f930d6ebf) it's barely understandable.

Done. I divided that commit in two, and modified the body of the commit description. Now you have 62cb9e06f5f72716c5ea794946ce10ff49be3274 and  98aba5517380266ef1a0f730c189c6c49c32de01.

File: src/api/app/helpers/webui/project_helper.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module Webui::ProjectHelper
+  # bento_only, but needed by obs_factory
```
Comment (hennevogel) [inline src/api/app/helpers/webui/project_helper.rb]: Can you move them out to a `Webui::ObsFactoryHelper` or something?

Comment (eduardoj) [reply src/api/app/helpers/webui/project_helper.rb]: Yep. I'll do it.

Comment (eduardoj) [reply src/api/app/helpers/webui/project_helper.rb]: Done. Please re-review.

**Pull request merged**